test/dynamo/test_misc.py

```diff
@@ -925,6 +925,314 @@ def forward(self, arg0_1: "f32[3][1]cpu", arg1_1: "f32[3][1]cpu", arg2_1: "f32[3
             cleanup_op("mylib::foo")
             del lib
 
+    def run_aot_eager(self, f, orig_args):
```

Comment:
Could we split out these tests to a new file? (something that isn't test/dynamo/test_misc.py?)